client/my-sites/stats/stats-redirect/index.tsx

```diff
@@ -73,8 +70,9 @@ const StatsRedirectFlow: React.FC< StatsRedirectFlowProps > = ( { children } ) =
 	}, [ dispatch, siteId, isLoadingNotices, purchaseNotPostponed ] );
 
 	// Render conditions (for readability).
+	// TODO: Determine the paywall redirection for specific routes.
 	const shouldRenderPaywall =
-		! isLoading && ! skipPaywallFlow && redirectToPurchase && siteSlug && canUserManageOptions;
+		false && ! isLoading && redirectToPurchase && siteSlug && canUserManageOptions;
```

Comment:
It seems the summary pages are all properly gated:

<img width="1108" alt="image" src="https://github.com/user-attachments/assets/f90f7fd6-6e82-4f61-a407-a21239acd077">


Let's revisit whether there are some pages that still need the paywall. If not, we'll be able to get rid of the legacy code to optimize bundle size.

Reply:
Tracked [here](https://github.com/Automattic/red-team/issues/139).